Pegasus/include/pegasus/Body.hpp

```diff
@@ -23,43 +25,87 @@ struct Body
      */
     struct Material
     {
-        PEGASUS_EXPORT Material();
+        Material();
 
         /**
          * @brief Sets mass of the body
          * @param mass mass
          */
-        PEGASUS_EXPORT void SetMass(double mass);
+        void SetMass(double mass);
 
         /**
          * @brief Sets body mass, and inverse mass equal to 0
          */
-        PEGASUS_EXPORT void SetInfiniteMass();
+        void SetInfiniteMass();
+
+        /**
+        * @brief Sets moment of inertia of the body
+        */
+        void SetMomentOfInertia(glm::mat3 momentOfInertia)
+        {
```

Comment:
It's basically just an instantiation of a single object, so it could be easily inlined, and the function is declared only for clarity, I don't really want to use an extra stack space and function call with this helper function. Other than that there is no reason.